modules/web-console/web-console-server/src/main/java/org/apache/ignite/console/web/controller/AccountController.java

```diff
@@ -107,7 +109,7 @@ public AccountController(AuthenticationManager authMgr, AccountsService accounts
 
         SecurityContextHolder.getContext().setAuthentication(authentication);
 
-        return ResponseEntity.ok().build();
+        return ResponseEntity.ok(new UserResponse(acc, req.isUserInRole(ROLE_PREVIOUS_ADMINISTRATOR)));
```

Comment:
Lets move "req.isUserInRole(ROLE_PREVIOUS_ADMINISTRATOR)" to separate utility method like:
public static boolean isBecomeUsed(req) { return req.isUserInRole(ROLE_PREVIOUS_ADMINISTRATOR)}

and use it in AccountController and GridGainAccountController